environment-ci.yml

```
@@ -11,7 +11,7 @@ dependencies:
   - nbformat>=5.1.2
   - pyyaml
   - toml
-  # - markdown-it-py~=1.0.0
+  - markdown-it-py[plugins]~=1.0
```

```suggestion
  - markdown-it-py~=1.0
  - mdit-py-plugins
```

Hey @mwouts, I'm surprised this does not throw an error; you can't have extras with Conda, but anyhow the `plugins` extra just adds `mdit-py-plugins`

---

Thanks @chrisjsewell for pointing this out. Yes it might be that the CI works just because we do a `pip install -e jupytext`... I'll have a look before merging this!